src/defaults/html.pipe.js

```diff
@@ -63,6 +64,7 @@ const htmlpipe = (cont, payload, action) => {
     .before(selecttest)
     .before(html).expose('html')
     .before(responsive)
+    .before(sanitize)
```

Comment:
once we modified the entire pipeline (including the HTL engine) to be based on JSDOM (see #337), should be move this after the `once()`  step?

Reply:
No, because this would make it impossible to have unsanitized output, which would severely reduce the power and expressiveness of the templates. As we generally assume developers know what they want, we shouldn't do this.